pallets/handles/README.md

```diff
@@ -0,0 +1,74 @@
+# Handles Pallet
+
+Creates human-readable, homoglyph attack resistent handles for MSA.
+
+## Summary
+
+Provides MSAs with an optional, but unique handle.
+The handles are able to be
+
+- **Base Handle:** The user's chosen handle string. It is not guaranteed to be unique.
+- **Suffix:** A suffix is a unique numeric value appended to a handle's canonical base to make it unique.
+- **Display Handle:** The base and the suffix together (`base`.`suffix`) is a unique identifier for a user.
+
+
+### UTF-8 Support
+
+Handles are able to have many allowed ranges of UTF-8.
+Some ranges, such as emoji, are currently not allowed.
+Due to the handling of homoglyphs, some handles will resolve to the same value.
+For example, while the display may have diacriticals, the handle is stored without diacriticals.
+So `Zoë` and `Zoe` would resolve to the same handle.
```

Comment:
I don't think it's entirely clear here that the diacriticals & homoglyphs are completely eliminated; all handles AND DISPLAY HANDLES are normalized to the canonical base.

Reply:
Updated some.